sdk/nodejs/tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "module": "ES2022",
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
+    "experimentalDecorators": true,
```

Comment:
TypeScript 5.0 was released in March with [stable support for decorators](https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#decorators). The experimental decorators is from a much older and less capable proposal.

Reply:
True, I need to update the typescript version though, I'll check that and see if I can fix it

Reply:
Resolved in [27477bc](https://github.com/dagger/dagger/pull/6159/commits/27477bc051006373f25a0ad00364ef1e6bc95bea)